components/suivi-form/livrables/stockage-refresh.js

```diff
@@ -0,0 +1,46 @@
+import {useState} from 'react'
+import PropTypes from 'prop-types'
+
+import colors from '@/styles/colors.js'
+
+const StockageRefresh = ({handleRefreshScan, stockageId}) => {
+  const [isScanRefreshing, setIsScanRefreshing] = useState(false)
+  const [errorMessage, setErrorMessage] = useState(null)
+
+  const handleClick = async () => {
+    try {
+      await handleRefreshScan(stockageId)
+      setIsScanRefreshing(true)
+    } catch (error) {
+      setErrorMessage(error.message)
+    }
+  }
+
+  return (
+    <div className='fr-grid-row'>
+      <button
+        type='button'
+        disabled={isScanRefreshing}
+        className='fr-btn fr-btn--sm fr-btn--secondary fr-btn--icon-left fr-icon-refresh-line'
+        onClick={() => handleClick()}
```

Comment:
La arrow fonction n'est pas utile ici.
```suggestion
        onClick={handleClick}
```